auditapi/VERSION.go

```diff
@@ -1,3 +1,3 @@
 package auditapi
 
-const SdkVersion = "2.0.3"
+const SdkVersion = "2.0.4-SNAPSHOT"
```

Comment:
`-SNAPSHOT` should be removed here.

Specifically,
1. Remove it in `develop`. (`2.0.4`)
2. Merge the `develop` branch into this branch.
3. Re-add and bump version in `develop` (aka `2.0.5-SNAPSHOT`)